### ISSUE TITLE
syb_nsql_nostatus attribute doesn't work [rt.cpan.org #20445]

### ISSUE THREAD

**Issue by mpeppler** [mpeppler/DBD-Sybase#5]
Migrated from [rt.cpan.org#20445](https://rt.cpan.org/Ticket/Display.html?id=20445) (status was 'new')

Requestors:
* nick.cabatoff@morganstanley.com

From  on 2006-07-13 14:03:17
:
```
When calling stored procedures using syb_nsql(), the result status of
the stored procedure should only be included in the results when
syb_nsql_nostatus is false.  However, the result status is included
regardless of the attribute's setting.  Stepping through the nsql code,
it looks like the problem is that you set

  	my $nostatus = $dbh->FETCH('syb_nsql_nostatus');

but never use it; instead, you say

  if ( $DBD::Sybase::syb_nsql_nostatus &&
			 $sth->{syb_result_type} == &CS_STATUS_RESULT ) {

When I stepped through the code, $DBD::Sybase::syb_nsql_nostatus was
undef regardless of the setting of the syb_nsql_nostatus attribute.

In other words, the above if statement should be written

  if ( $nostatus && $sth->{syb_result_type} == &CS_STATUS_RESULT ) {


```

**Comment by mpeppler** [comment]
This has been fixed a long time ago...
